### PR TITLE
6.11.0

### PR DESCRIPTION
**Pull request by iche033**
Signed-off-by: Ian Chen <ichen@osrfoundation.org>

# 🎈 Release

Preparation for 6.11.0 release.

Comparison to 6.10.0: https://github.com/gazebosim/gz-sim/compare/ignition-gazebo6_6.10.0...prep_6.11.0

## Checklist
- [x] Asked team if this is a good time for a release
- [x] There are no changes to be ported from the previous major version
- [x] No PRs targeted at this major version are close to getting in
- [x] Bumped minor for new features, patch for bug fixes
- [x] Updated changelog
- [x] ~~Updated migration guide (as needed)~~
- [x] Link to PR updating dependency versions in appropriate repository in [gazebo-release](https://github.com/gazebo-release) (as needed): <LINK> https://github.com/gazebo-release/gz-sim6-release/pull/24

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,91 @@
...
+    * [Pull request #1549](https://github.com/gazebosim/gz-gazebo/pull/1549)
+
+1. Quick start dialog
+    * [Pull request #1536](https://github.com/gazebosim/gz-gazebo/pull/1536)
+
+1. Quiet libSDFormat console on --verbose 0
+    * [Pull request #1621](https://github.com/gazebosim/gz-gazebo/pull/1621)
+
+1. Add Ackermann Steering system (backport from Fortress)
+    * [Pull request #1613](https://github.com/gazebosim/gz-gazebo/pull/1613)
```
Comment (chapulina) [inline Changelog.md:36]: This entry could be removed

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,91 @@
...
+    * [Pull request #1607](https://github.com/gazebosim/gz-gazebo/pull/1607)
+
+1. Change CODEOWNERS and maintainer to Michael
+    * [Pull request #1644](https://github.com/gazebosim/gz-gazebo/pull/1644)
+
+1. Replace pose in ViewAngle with GzPose
+    * [Pull request #1641](https://github.com/gazebosim/gz-gazebo/pull/1641)
+
+1. Fix loading worlds from CLI
+    * [Pull request #1627](https://github.com/gazebosim/gz-gazebo/pull/1627)
```
Comment (chapulina) [inline Changelog.md:24]: This could be grouped with the quick start dialog

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,91 @@
...
+    * [Pull request #1590](https://github.com/gazebosim/gz-gazebo/pull/1590)
+
+1. Fix compilation of scene broadcaster test
+    * [Pull request #1599](https://github.com/gazebosim/gz-gazebo/pull/1599)
+
+1. Helper function to get an entity from an entity message
+    * [Pull request #1595](https://github.com/gazebosim/gz-gazebo/pull/1595)
+
+1. Fix compilation of scene broadcaster test
+    * [Pull request #1599](https://github.com/gazebosim/gz-gazebo/pull/1599)
```
Comment (chapulina) [inline Changelog.md:63]: This is repeated

**Review by chapulina — COMMENTED** [review]
Need to update the release repo with the new dependency versions:

https://github.com/gazebosim/gz-sim/compare/ignition-gazebo6_6.10.0...prep_6.11.0#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20a

File: Changelog.md [hunk 4]
```diff
@@ -1,5 +1,91 @@
 ## Ignition Gazebo 6.x
 
+### Gazebo Sim 6.11.0 (2022-08-17)
+
+1. Add support for specifying log record period
+    * [Pull request #1636](https://github.com/gazebosim/gz-gazebo/pull/1636)
```
Comment (jennuine) [inline Changelog.md:6]: All these urls need to be updated to `gz-sim`

```suggestion
    * [Pull request #1636](https://github.com/gazebosim/gz-sim/pull/1636)
```

Comment (iche033) [reply Changelog.md]: fixed. [2f68bc4](https://github.com/gazebosim/gz-sim/pull/1656/commits/2f68bc4253e15d0c0a89dcde8fc21342bbcd3545)

File: Changelog.md (hunk 3 above)
Comment (iche033) [reply Changelog.md]: removed. [2f68bc4](https://github.com/gazebosim/gz-sim/pull/1656/commits/2f68bc4253e15d0c0a89dcde8fc21342bbcd3545)

File: Changelog.md (hunk 2 above)
Comment (iche033) [reply Changelog.md]: grouped. [2f68bc4](https://github.com/gazebosim/gz-sim/pull/1656/commits/2f68bc4253e15d0c0a89dcde8fc21342bbcd3545)

File: Changelog.md (hunk 1 above)
Comment (iche033) [reply Changelog.md]: removed. [2f68bc4](https://github.com/gazebosim/gz-sim/pull/1656/commits/2f68bc4253e15d0c0a89dcde8fc21342bbcd3545)

**Pull request merged**